jrnl/plugins/text_exporter.py

```diff
@@ -35,6 +35,9 @@ def write_file(cls, journal, path):
                 return f"[Journal exported to {path}]"
         except IOError as e:
             return f"[{ERROR_COLOR}ERROR{RESET_COLOR}: {e.filename} {e.strerror}]"
+        except RuntimeError as e:
+            os.remove(path)
```

Comment:
Why are we deleting the path here? This seems like new behavior.

Reply:
It seemed like a good idea to cleanup the empty output file after an exception was raised.

Reply:
Currently, if a directory doesn't exist, a file is opened instead which remains after the command terminates.

Reply:
If you're worried about this applying too generally to every RuntimeError, we could also create a more specific error like ExportError. That would be used to signal that the empty or partially written file needs to be deleted and the error message should be returned as a string.

Reply:
I can understand the rationale for cleaning this path up, but I'd prefer that jrnl doesn't do this. A user might be exporting to a directory that already has other data in it, and this could remove that data. Better to leave the data and let the user deal with it appropriately.

Reply:
Okay, sounds good. I'll remove that then.